From f25eb61c64fb8744698702d0adb038ef7c255a05 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 9 Feb 2016 14:59:27 +0000
Subject: [PATCH 201/462] MANTIS 0037350 initial: integration of EZ-Access
 device for KIS kiosk

removed modules
 Massai\cpp\Keyboard\KeyboardDriverService
 Massai\cpp\Keyboard\Keyboard
 Massai\cpp\Keyboard\KeyboardDriverInstaller
 Massai\cpp\Keyboard\KeyboardTest

more parts of wdk necessary in build environment

git-svn-id: svn://localhost/SelfServiceBuild/trunk@307 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/cpp/makefile.mak | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/SelfServiceBuild/Build/cpp/makefile.mak b/SelfServiceBuild/Build/cpp/makefile.mak
index 061c293..32669f0 100644
--- a/SelfServiceBuild/Build/cpp/makefile.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.mak
@@ -150,10 +150,10 @@ PLATFORM_DIRECTORIES= \
  Massai\cpp\BillingService\static \
  Massai\cpp\specialElectronics \
  Massai\cpp\Illumination \
- Massai\cpp\Keyboard\KeyboardDriverService \
- Massai\cpp\Keyboard\Keyboard \
- Massai\cpp\Keyboard\KeyboardDriverInstaller \
- Massai\cpp\Keyboard\KeyboardTest \
+# Massai\cpp\Keyboard\KeyboardDriverService \
+# Massai\cpp\Keyboard\Keyboard \
+# Massai\cpp\Keyboard\KeyboardDriverInstaller \
+# Massai\cpp\Keyboard\KeyboardTest \
 
 PLATFORM_INSTALLER_DIRECTORIES= \
  $(INSTALLER_DIRECTORIES)
-- 
2.41.0.windows.1

